src/commands/MigrationShowCommand.ts

```diff
@@ -0,0 +1,70 @@
+import {createConnection} from "../index";
+import {ConnectionOptionsReader} from "../connection/ConnectionOptionsReader";
+import {Connection} from "../connection/Connection";
+import * as process from "process";
+import * as yargs from "yargs";
+const chalk = require("chalk");
+
+/**
+ * Runs migration command.
+ */
+export class MigrationShowCommand implements yargs.CommandModule {
+
+  command = "migration:show";
+  describe = "Show all migrations and whether they have been run or not";
+
+  builder(args: yargs.Argv) {
+    return args
+      .option("connection", {
+        alias: "c",
+        default: "default",
+        describe: "Name of the connection on which run a query."
+      })
+      .option("transaction", {
```

Comment:
It's probably not necessary if we're just going to show migrations without running them.